docs/create_embed.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+set -euo pipefail
+
+TOKEN=${TOKEN:?"TOKEN env var not set"}
+FILE=${1:?usage: create_embed.sh <filename.\{ts,js,python,go\}>}
```

Comment:
Should we also add support for .mjs and .py?

Is it possible to combine this script with the quickstart-specific one, so that it creates a normal embed or a quickstart embed based on a command-line parameter?